### ISSUE TITLE
debug errors/stack trace about Locus not being installed

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#10385]
Another one would be, that most logs I receive on support start with warnings about Locus not being installed logging the whole stacktracke. IMHO this is also kind of spam. Couldn't this be logged just as a one line warning and maybe even only in debug mode?

_Originally posted by @Lineflyer in https://github.com/cgeo/cgeo/discussions/10383_

**Comment by moving-bits** [comment]
This comes from the Locus API, which checks seven different app package names for Locus, and logs the ones not found, including the stacktrace, see LocusUtils.kt in their package:

```
        for (pn in packageNames) {
            try {
                val packageInfo = pm.getPackageInfo(pn, 0)
                val appInfo = pm.getApplicationInfo(pn, 0)
                val lv = createLocusVersion(ctx, appInfo.packageName)
                if (lv != null) {
                    versions.add(lv)
                }
            } catch (ignored: PackageManager.NameNotFoundException) {
                Logger.logD(TAG, "getAvailableVersions($ctx), " +
                        "e: $ignored")
                ignored.printStackTrace()
            }
        }
```

Don't know if there is a way to suppress such messages except for the Locus team removing the logging sequence.

**Comment by bekuno** [comment]
The 'ignored.printStackTrace()' should not be there, I would say.

**Comment by eddiemuc** [comment]
This error is already fixed on master branch of LocusUtils:

https://github.com/asamm/locus-api/blob/f40ef9276fc6a1c3ee8bedf4d6e3c6fdb71ae9d7/locus-api-android/src/main/java/locus/api/android/utils/LocusUtils.kt#L154-L159

**Comment by eddiemuc** [comment]
...and I assume this is included in latest release 2 days ago:

![image](https://user-images.githubusercontent.com/6909759/115035565-fc661e80-9ecc-11eb-99ac-af928e97f908.png)

**Comment by bekuno** [comment]
correct, in commit https://github.com/asamm/locus-api/commit/3f5f8808385b565eac7c29d50ad00c8365a82a84
which is included in 0.9.32
